### PR TITLE
Augment new starter content

### PR DESCRIPTION
**Pull request by jonathanglassman**
Augment new starter content:

- Background page - add links to blogs explaining the background of GOV.UK Account
- How GOV.UK Account works - summarise different parts of the architecture and links to more detailed information
- Get access to required apps and information - add in information
- Created section telling new starters what tools they need to install

Also changed "GOV.UK Accounts" to "GOV.UK Account" in two sections

As per https://trello.com/c/uB19t27Z/559-review-and-augment-new-starter-content-in-team-manual

### CODE REVIEW

File: source/new-starter/access/index.html.md.erb [hunk 1]
```diff
@@ -1,10 +1,64 @@
...
 
-# Access
+# Get access to required apps and information
 
-List of all the things you should have access to and how to get that access
+When you join the GOV.UK Account team, you must get access to all the required apps and information you need.
+
+## Complete the GOV.UK new starter process
+
+If you are new to GDS or the GOV.UK program, you must complete the GOV.UK new starter process.
```
Comment (barrucadu) [inline source/new-starter/access/index.html.md.erb]: "programme" (and below)

File: source/new-starter/access/index.html.md.erb [hunk 2]
```diff
@@ -1,10 +1,64 @@
...
+- [Zendesk](https://docs.publishing.service.gov.uk/manual/zendesk.html)
+- [`govuk-accounts` Slack channel](https://app.slack.com/client/T8GT9416G/C012GEZBZM0)
+- [`govuk-accounts-tech` Slack channel](https://app.slack.com/client/T8GT9416G/C011Y5SAY3U)
+
+## Get access to GOV.UK PaaS and Notify
+
+Ask the GOV.UK Account tech lead or lead developer for access to the following:
+
+- [CloudFoundry](https://www.cloudfoundry.org/)
+- [GOV.UK PaaS](https://www.cloud.service.gov.uk/) as a [space developer](https://docs.cloud.service.gov.uk/orgs_spaces_users.html#space-developer)
```
Comment (barrucadu) [inline source/new-starter/access/index.html.md.erb]: The PaaS is CloudFoundry

File: source/new-starter/how-it-works/index.html.md.erb [hunk 3]
```diff
@@ -1,8 +1,56 @@
...
+# How GOV.UK Account works
+
+The core GOV.UK account architecture includes:
+
+- an authentication service
+- an attribute store
+- a broker
+- a preferences and consent management service
+- a trust framework
+- a fraud protection and monitoring service
```
Comment (barrucadu) [inline source/new-starter/how-it-works/index.html.md.erb]: We don't actually have any of these other than the first two implemented yet

File: source/new-starter/install-tools/index.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+title: Install developer tools
+weight: 23
+last_reviewed_on: 2020-12-10
+review_in: 6 months
+---
+
+# Install developer tools
+
+The GOV.UK Accounts team uses developer tools.
```
Comment (barrucadu) [inline source/new-starter/install-tools/index.html.md.erb]: Should be "Account" for consistency

File: source/new-starter/install-tools/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+When you join the GOV.UK Account team, you must install these tools.
+
+## Install developer tools
+
+All developers must install [`gds-cli`, the Government Digital Service Command Line Interface].(https://github.com/alphagov/gds-cli).
```
Comment (barrucadu) [inline source/new-starter/install-tools/index.html.md.erb]: Broken link, change `].(` to `](`

File: source/new-starter/install-tools/index.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+You use this tool to connect to the VPN and update secrets.
+
+To install `gds-cli`, see the [GOV.UK Developer documentation on installing GDS tooling](https://docs.publishing.service.gov.uk/manual/get-started.html#3-install-gds-tooling) for more information.
+
+## Install backend developer tools
```
Comment (barrucadu) [inline source/new-starter/install-tools/index.html.md.erb]: `gds-cli` is also a backend tool

File: source/new-starter/how-it-works/index.html.md.erb (hunk 3 above)
Comment (jonathanglassman) [reply source/new-starter/how-it-works/index.html.md.erb]: have amended to be clear about current vs future architecture

File: source/new-starter/install-tools/index.html.md.erb (hunk 4 above)
Comment (jonathanglassman) [reply source/new-starter/install-tools/index.html.md.erb]: Have corrected in this and other files

File: source/new-starter/access/index.html.md.erb [hunk 7]
```diff
@@ -1,10 +1,63 @@
...
+
+1. Sign in to the [GDS IT & Estates Web Help Desk](https://gdshelpdesk.digital.cabinet-office.gov.uk/helpdesk/WebObjects/Helpdesk.woa).
+1. In __Request Type__, select __GDS IT Requests__ > __People & accounts__ > __Other Account Requests__.
+1. In __Request Detail__, enter "Please apply gds-027-govuk-accounts role to my google account."
+1. For __Account type__, select LDAP.
+1. In __For Google changes__, select __Account correction/change__.
+1. In __Is this request for you?__, select __Yes__.
+1. Select your __Location__ and __Room__, and select __Save__.
+
+When IT tell you the request is complete, check that you can access the [GOV.UK Account Splunk dashboard](https://gds.splunkcloud.com/en-US/app/gds-006-govuk/dashboards).
```
Comment (brucebolt) [inline source/new-starter/access/index.html.md.erb]: This links to the GOV.UK dashboards, which you won't get access to by requesting the `gds-027-govuk-accounts` role from IT.
```suggestion
When IT tell you the request is complete, check that you can access the [GOV.UK Account Splunk dashboard](https://gds.splunkcloud.com/en-US/app/gds-027-govuk-accounts/dashboards).
```

File: source/new-starter/install-tools/index.html.md.erb [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+- `gds-cli`, the Government Digital Service Command Line Interface (CLI)](https://github.com/alphagov/gds-cli) to connect to the VPN and update secrets
+- fly, a CLI to interact with Concourse
+- the CloudFoundry CLI to interact with our servers on the GOV.UK PaaS
+
+When you join the GOV.UK Account team, you must install these tools.
+
+## Install the gds-cli
+
+The `gds-cli` is a CLI to connect to the VPN and update secrets.
```
Comment (brucebolt) [inline source/new-starter/install-tools/index.html.md.erb]: Can we link to (or add docs) on how to connect to the VPN and update secrets using `gds-cli`?  I'm not sure we've covered this, and a new starter probably won't know how to do this.

File: source/new-starter/access/index.html.md.erb [hunk 9]
```diff
@@ -1,10 +1,63 @@
...
 
-# Access
+# Get access to required apps and information
 
-List of all the things you should have access to and how to get that access
+When you join the GOV.UK Account team, you must get access to all the required apps and information you need.
+
+## Complete the GOV.UK new starter process
+
+If you are new to GDS or the GOV.UK programme, you must complete the GOV.UK new starter process.
```
Comment (brucebolt) [inline source/new-starter/access/index.html.md.erb]: Should we make it clear that only tech people are required to do these tasks?

File: source/new-starter/access/index.html.md.erb [hunk 10]
```diff
@@ -1,10 +1,63 @@
...
+1. For __Account type__, select LDAP.
+1. In __For Google changes__, select __Account correction/change__.
+1. In __Is this request for you?__, select __Yes__.
+1. Select your __Location__ and __Room__, and select __Save__.
+
+When IT tell you the request is complete, check that you can access the [GOV.UK Account Splunk dashboard](https://gds.splunkcloud.com/en-US/app/gds-006-govuk/dashboards).
+
+## Check access to the GOV.UK Account Grafana dashboard
+
+Check that you have access to the [GOV.UK Account Grafana dashboard](https://grafana-paas.cloudapps.digital/gov-uk-accounts).
```
Comment (brucebolt) [inline source/new-starter/access/index.html.md.erb]: Can we add instructions on what to do or who to speak to if you don't have access?

Comment (brucebolt) [inline source/new-starter/access/index.html.md.erb]: Also, the link doesn't seem to work for me.  I think it should be https://grafana-paas.cloudapps.digital/d/_g_9aRoMk/gov-uk-accounts.

File: source/new-starter/access/index.html.md.erb (hunk 9 above)
Comment (jonathanglassman) [reply source/new-starter/access/index.html.md.erb]: Given that the intended user of the team manual is developers, and this is stated on the manual landing page, i think we're ok.

Comment (jonathanglassman) [reply source/new-starter/access/index.html.md.erb]: But ok i'll make it clear again

File: source/new-starter/access/index.html.md.erb (hunk 10 above)
Comment (jonathanglassman) [reply source/new-starter/access/index.html.md.erb]: Good point. Who should you speak to?

File: source/new-starter/install-tools/index.html.md.erb (hunk 8 above)
Comment (jonathanglassman) [reply source/new-starter/install-tools/index.html.md.erb]: Will add links to new team manual docs on rotating secrets and gov.uk developer docs on connecting to the VPn

**Comment by huwd** [comment]
Oh we might want to rename a few of these commits perhaps?
Sorry I feel I've become the commit pedant?

The current names seem to tell me more about how we worked on this, whereas they're more useful in a long long if we can see what a particularly commit changed?

They could all be squashed down into a single commit which matches the PR title "Augment new starter content" perhaps?

**Comment by jonathanglassman** [comment]
> Oh we might want to rename a few of these commits perhaps?
> Sorry I feel I've become the commit pedant?
> 
> The current names seem to tell me more about how we worked on this, whereas they're more useful in a long long if we can see what a particularly commit changed?
> 
> They could all be squashed down into a single commit which matches the PR title "Augment new starter content" perhaps?

done

**Review by huwd — APPROVED** [review]

**Pull request merged**